src/AppInstallerCLICore/Argument.cpp

```diff
@@ -110,6 +110,7 @@ namespace AppInstaller::CLI
         args.push_back(ForType(Args::Type::RainbowStyle));
         args.push_back(ForType(Args::Type::RetroStyle));
         args.push_back(ForType(Args::Type::VerboseLogs));
+        args.emplace_back("disable-interactivity", NoAlias, Args::Type::DisableInteractivity, Resource::String::DisableInteractivityArgumentDescription, ArgumentType::Standard, false);

```

Comment:
Should this be a flag?  You use it like one in PromptFlow, so I think yes.

Reply:
You're right. Changed to flag.